integration-tests/src/test/resources/admin_api/readonlyrest_first_update.yml

```diff
@@ -1,5 +1,7 @@
 readonlyrest:
 
+  prompt_for_basic_auth: true
```

Comment:
I understand that we should assert 403 instead of 401 currently? IMO we should do that. 
Obviously, we should have a test when we check that when `prompt_for_basic_auth: true` is set and the request is forbidden, the HTTP 401 is returned instead of 403. But in the rest of tests, we should use 403.